v2/internal/project/project.go

```diff
@@ -88,6 +89,23 @@ type Project struct {
 	// Garble
 	Obfuscated bool   `json:"obfuscated"`
 	GarbleArgs string `json:"garbleargs"`
+
+	// Frontend directory
+	FrontendDir string `json:"frontend:dir"`
+}
+
+func (p *Project) GetFrontendDir() string {
+	if filepath.IsAbs(p.FrontendDir) {
+		return p.FrontendDir
+	}
+	return filepath.Join(p.Path, p.FrontendDir)
```

Comment:
Since we are exposing `p.Path` to the user `p.FrontendDir` is not always relative to where `wails.json` is found. It might be relative to that or relative to `projectdir`.
Would it be more consistent and easier to reason about this folder if it is always relative to `wails.json` and `projectdir` is also relative to `wails.json`?
So every folder defined in `wails.json` would be relative to the location if the file itself.

Reply:
Yes, I agree - every relative path in `wails.json`. Right now `p.Path` *is* where `wails.json` is (after I remove "projectdir") so I think it's good for now. I think eventually, `project.Load()` will be path-less and use cwd by default. It will traverse the filesystem to find it. Once found, it will load and write this location to the project data internally. This method will calculate based off that.

Reply:
Yeah, with removing projectdir that's fine.